wuvt/templates/donate/premium_form.html

```diff
@@ -67,6 +67,11 @@
                 value="ship" />
             Yes, ship them to me (+${{ config.DONATE_SHIPPING_COST }}, US only)
         </label>
+        <label>
+            <b>Note:</b> we are unable to guarantee premium availability if pledges
+            are made outside of Radiothon week. If you make a recurring monthly
+            donation, you will only be sent premiums for the first month.
+        </label>
 
         <div id="premium_fields">
```

Comment:
Can you move that text into this `<div>` so that it only shows up if the user specifies that they want premiums?